src/components/CreateListing/CreateListing.tsx

```diff
@@ -148,16 +148,23 @@ export const CreateListing = () => {
           swapper: addressBech32(account.address!),
         };
 
+        const tags = {
+          [env.NEXT_PUBLIC_DAPP_ID]: {
+            startDate: startDate !== "" ? startDate : new Date().toISOString(),
+            expiryDate,
+          },
+          [env.NEXT_PUBLIC_DAPP_ID +
+          SWAP_TAG +
+          selectedOffered.option.toLowerCase()]: "",
+          [env.NEXT_PUBLIC_DAPP_ID +
+          SWAP_TAG +
+          selectedDesired.option.toLowerCase()]: "",
```

Comment:
I don't understand these tags, why are they empty on the value?

Using something like 

```
const tagNameWithSemanticMeaning = `${env.NEXT_PUBLIC_DAPP_ID}-${SWAP_TAG}-${selectedOffered.option.toLowerCase()}`

const tags = {
  // ...
 // Comment saying why we are tagging the contract with this 
  tagNameWithSemanticMeaning: ""
}
```

Reply:
I used the tags to search by tokens in the listing. Then I inclued on `client.getContracts()` the token name and filter by tag. It's empty because I have no info to add...